navfn/src/navtest.cpp

```diff
@@ -199,6 +199,8 @@ int main(int argc, char **argv)
     }
   }
 
+  free(cmap);
```

Comment:
Question:
Is there the release function corresponding to readPGM()?

Reply:
Does not exist.
readPGM itself is defined in this source.
Not like a general library.

Reply:
OK.